releasenotes/notes/fix-websocket-durations-0e0b07916912294d.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixes issue with websocket.receive span not closing exactly when another websocket.receive span was opened. 
+    Also fixes duration of handshake span such that span closes when connection is upgraded. 
+    Minor fix where websocket.close parent should be the handshake span when configuration is disabled.
```

```suggestion
fixes:
  - |
    tracing: Fixes issue with ``websocket.receive`` span not closing exactly when another ``websocket.receive`` span was opened. 
  - |
    tracing: Fixes duration of websocket handshake span such that the handshake span closes when the connection is upgraded. 
  - |
    tracing: Fix where ``websocket.close`` parent should be the handshake span when configuration is disabled.
```


We can define multiple release note entries in a single file